src/main/java/com/hubspot/jinjava/lib/filter/PrettyPrintFilter.java

```diff
@@ -48,7 +50,16 @@ public Object filter(Object var, JinjavaInterpreter interpreter, String... args)
     ) {
       varStr = Objects.toString(var);
     } else {
-      varStr = new POJONode(var).toPrettyString();
+      try {
+        varStr =
+          interpreter
+            .getConfig()
+            .getObjectMapper()
```

Comment:
Thoughts on registering the module here so we scope the change to the object mapper to the pretty print filter? I think my concern here is if changing the mapper on the config level will inadvertently change something else

Reply:
Good idea. I'll update.

Reply:
I don't think registering here is the right move. This will register the module on every call to the `PrettyPrintFilter`.

I need to look through the PR to figure out where the right place to do it would be.

Reply:
Updated to remove module registration, but the remaining changes are still needed to support using the interpreter's config's `ObjectMapper`.

Reply:
👍